src/main/java/com/inqoo/trainingservice/app/models/Subcategory.java

```diff
@@ -11,10 +11,13 @@
     private String name;
     @Column(length = 300)
     private String description;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
maybe we can try without it - but maybe in next step

Reply:
Ok.